configs/spmd/knn.json

```diff
@@ -0,0 +1,27 @@
+{
+    "INCLUDE": ["../common/sklearn.json", "../regular/knn.json"],
+    "PARAMETERS_SETS": {
+        "spmd knn parameters": {
+            "algorithm": {
+                "estimator_params": {
+                    "algorithm": "brute",
+                    "metric": "minkowski",
+                    "p": 2,
```

Comment:
1 should also still be compatible with spmd knn if aligning with regular would be preferred

Reply:
`p=1` is more uncommon case for kNN, I guess it can be added in case of the config extension.